test/e2e/app-dir/app/app/report-web-vitals/reporter.js

```diff
@@ -0,0 +1,17 @@
+'use client'
+import { useReportWebVitals } from 'next/client'
```

Comment:
`next/client` is not an api we currently expose for usage. We'll need to come up with a better name for which module exports this.